arangod/V8Server/v8-voccursor.cpp

```diff
@@ -538,7 +538,7 @@ struct V8Cursor final {
     if (self == nullptr) {
       TRI_V8_RETURN(v8::Undefined(isolate));
     }
-    TRI_V8_RETURN(v8::Integer::New(isolate, self->_cursorId));
+    TRI_V8_RETURN(v8::Integer::New(isolate, static_cast<uint32_t>(self->_cursorId)));
```

Comment:
This will lead to truncation of larger ids (same for the old version).
We should rather return the cursor id as a string, as we do for other potentially big integers.
We may need to adjust callers of this function to handle the adjusted return type.